src/meta/nodeExt/nodeExt.ts

```diff
@@ -12,10 +11,11 @@ export enum NodeExtType {
   node = 'node',
 }
 
-export type NodeExt<Datum = object> = {
-  readonly uuid: string
+export type NodeExt<Props = unknown, Value = unknown> = {
```

Comment:
<img width="767" alt="Screenshot 2023-11-10 at 10 22 58" src="https://github.com/openforis/fra-platform/assets/5508251/d87caaaf-8d07-4314-b563-266b70a0fc2a">

Reply:
Initially I implemented both as `{}` but because of these, I had to change it:

<img width="853" alt="image" src="https://github.com/openforis/fra-platform/assets/5508251/4b455dd6-892b-437d-ab96-e473a78f2866">